Week-6/3sum.js

```diff
@@ -0,0 +1,33 @@
+function threeSum(arr, target) {
+  let result = "";
+
+  for (let i = 0; i < arr.length - 2; i++) {
+    for (let j = 1; j < arr.length - 1; j++) {
+      for (let k = j + 1; k < arr.length; k++) {
+        if (arr[i] + arr[j] + arr[k] === target) {
```

Comment:
@rajeshvlega9 It seems the question is to check for target as well as closest to it. Can you solve it in a better way

Please see this approach for reference - https://www.geeksforgeeks.org/find-a-triplet-that-sum-to-a-given-value/

Reply:
Hi Rajesh
made the changes. 
![image](https://user-images.githubusercontent.com/41244432/187253360-53841fdd-481f-4660-9b7b-ea73450dac97.png)